### PR TITLE
Everything v9.12.0 ready

### PR DESCRIPTION
**Pull request by S1SYPHOS**
- Added new / modified stylesheets
- Added new / modified language definitions
- Updated some functions
- Included bash script for easier lang def update
- Fixed headings, links, typos

### CODE REVIEW

**Comment by S1SYPHOS** [comment]
Full credits for 9655ad7 and d22a5d7 to Vladimir Jimenez aka @allejo - great work!

Language definitions so far up to 9.11.0, only one left: MikroTik RouterOS Scripting language by [Ivan Dementev](https://github.com/DiVAN1x).

File: composer.json [hunk 1]
```diff
@@ -10,5 +10,17 @@
...
-	}
+	},
+	"authors": [
+		{
+			"name": "Geert Bergman",
+			"homepage": "http://www.scrivo.org/",
+			"role": "Project Author"
+		},
+		{
+			"name": "Vladimir Jimenez",
```
Comment (allejo) [inline composer.json]: I added myself on here for the fork that I submitted to Packagist, I don't think I should belong in the official package.

File: composer.json [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
+    "psr-0": { "Highlight\\": "" }
+  },
+  "authors": [
+    {
+      "name": "Geert Bergman",
+      "homepage": "http://www.scrivo.org/",
+      "role": "Project Author"
+    },
+    {
+      "name": "Vladimir Jimenez",
```
Comment (allejo) [inline composer.json]: I added myself to the fork I submitted to Packagist, however I shouldn't belong in the official package.

Comment (allejo) [reply composer.json]: oh no, that's fair. my line of thinking was that the project author should decide who to list under "authors" and not just be snuck in there in a PR

Comment (S1SYPHOS) [reply composer.json]: True, but honestly I thought of it as "suggestion" 🙂

Comment (allejo) [reply composer.json]: that makes sense. sorry if I came off the wrong way 😅

Comment (S1SYPHOS) [reply composer.json]: 🐪 alright!

**Comment by allejo** [comment]
@S1SYPHOS, I was wondering if you'd object to letting me merge in changes from [stakx-io/highlight.php](https://github.com/stakx-io/highlight.php) and then rebasing the rest of your changes on top of those? Just because a lot of the changes you incorporated were from my fork.

**Comment by S1SYPHOS** [comment]
I was just about to close this PR, merge in your fork and then adding my changes atop.

**Pull request closed without merging**